.jshintrc

```diff
@@ -30,5 +29,5 @@
   "white": false,
   "eqnull": true,
   "esnext": true,
-  "unused": "vars"
+  "unused": true
```

Comment:
TBH I'm not sure, nor do I understand the motivation for the setting as it is in the first place. That said, you're at the helm, revert for this is on the way!